.shippable.yml

```diff
@@ -48,7 +48,7 @@ build:
     - _make CFG_TA_GPROF_SUPPORT=y CFG_FTRACE_SUPPORT=y CFG_SYSCALL_FTRACE=y CFG_ULIBS_MCOUNT=y
     - _make CFG_SECURE_DATA_PATH=y
     - _make CFG_REE_FS_TA_BUFFERED=y
-    - _make PLATFORM=vexpress-qemu_armv8a CFG_ARM64_core=y CFG_CORE_ASLR=y
+    - _make PLATFORM=vexpress-qemu_armv8a CFG_ARM64_core=y CFG_CORE_ASLR=y CFG_SCTLR_ALIGNMENT_CHECK=y
```

Comment:
How? Commit "Disable alignment check by default" sets it to `n` by default and I don't see that vexpress sets it?

Reply:
`CFG_SCTLR_ALIGNMENT_CHECK ?= y` since aaaf00a257488d2b6fbf8a4e842e1333529c704c and unmodified since.
It is not set from any **build.git** makefiles.

Reply:
The change you are commenting on is in patch "[Review] Disable alignment check by default" which updates "[Disable alignment check by default](https://github.com/OP-TEE/optee_os/pull/3693/commits/11e0b15fb1f8fbcc6b72bbed497aa7bc1d07cd40)" which sets `CFG_SCTLR_ALIGNMENT_CHECK ?= n` globally ;-) As mentioned the libgcc_eh.a library is not compiled with `-mstrict-align`.

Reply:
sorry. discard my comment :confused: